conan/conanfile.txt

```diff
@@ -4,19 +4,19 @@ gtest/1.8.1@bincrafters/stable
 librdkafka/0.11.4-dm2@ess-dmsc/stable
 spdlog-graylog/1.2.1-dm12@ess-dmsc/stable
 epics/3.16.1-4.6.0-dm6@ess-dmsc/stable
-streaming-data-types/c45d2ec@ess-dmsc/stable
+streaming-data-types/fb4d847@ess-dmsc/stable
```

Comment:
I've updated the streaming-data-types version to this on the dev env confluence page.

Reply:
Ok.